site/monitoring.xml

```diff
@@ -0,0 +1,105 @@
+      users.
+    </p>
+    <doc:section name="system-metrics">
+      <doc:heading>System Metrics</doc:heading>
+      <ul>
+        <li>CPU (idle, user, system, iowait)</li>
+        <li>System load average (<code>/proc/loadavg</code>)</li>
+        <li>Memory (free, cached, buffered)</li>
+        <li>Disk I/O (reads &amp; writes per unit time, I/O wait percentages)</li>
+        <li>Free Disk Space</li>
+        <li>File descriptors used by <code>beam.smp</code> vs available</li>
+        <li>Network throughput (bytes received, bytes sent)</li>
+        <li><a href="https://www.kernel.org/doc/Documentation/sysctl/vm.txt">VM</a> statistics (dirty page flushes, writeback volume)</li>
+      </ul>
+    </doc:section>
+    <doc:section name="rabbitmq-metrics">
+      <doc:heading>RabbitMQ Metrics</doc:heading>
+      <p>
+        The RabbitMQ <a href="management.html">management plugin</a> provides a starting point for
+        monitoring RabbitMQ metrics. One limitation, however, is that only up to one day's worth of
+        metrics are stored. Storing historical metrics can be an important tool to determine the root
+        cause of issues affecting your users or to plan for future capacity.
+      </p>
+      <p>
+        RabbitMQ metrics are made available through the <a href="management.html#http-api">HTTP API</a>
+        or via several <a href="man/rabbitmqctl.1.man.html"><code>rabbitmqctl</code></a> commands. It is
+        recommended to collect metrics at 60 second intervals because more frequent collection may place
+        too much load on the RabbitMQ server and negatively affect performance.
+      </p>
+      <p>
+        <table>
+          <thead>
+            <tr><td>Metric</td><td>JSON field name</td></tr>
+          </thead>
+          <tbody>
+            <tr><td>Memory</td><td><code>memory</code></td></tr>
+            <tr><td>Queued Messages</td><td><code>messages</code></td></tr>
+            <tr><td>Un-acked Messages</td><td><code>messages_unacknowledged</code></td></tr>
+            <tr><td>Messages Published</td><td><code>message_stats.publish</code></td></tr>
+            <tr><td>Message Publish Rate</td><td><code>message_stats.publish_details.rate</code></td></tr>
+            <tr><td>Messages Delivered</td><td><code>message_stats.deliver_get</code></td></tr>
+            <tr><td>Message Delivery Rate</td><td><code>message_stats.deliver_get.rate</code></td></tr>
+            <tr><td>Other Message Stats</td><td><code>message_stats.*</code> (see <a href="https://rawcdn.githack.com/rabbitmq/rabbitmq-management/master/priv/www/doc/stats.html">this document</a>)</td></tr>
+          </tbody>
+        </table>
+      </p>
+    </doc:section>
+    <doc:section name="third-party-monitoring-tools">
+      <doc:heading>Third Party Monitoring Tools</doc:heading>
+      <p>
+        The following is an alphabetized list of third-party tools to collect RabbitMQ metrics. These
```

Comment:
"alphabetised"? (RabbitMQ documentation is deeply rooted in British English  :-) ).